test/blackbox-tests/test-cases/pkg/ocamlformat/dune

```diff
@@ -1,3 +1,3 @@
 (cram
- (deps helpers.sh)
+ (deps helpers.sh %{bin:curl})
```

Comment:
Does every test really depend on curl?

Reply:
Not every test. I can make it more fine grained.

Reply:
Not completely necessary, but it is an improvement IMO. I just found it surprising that there are no tests that don't use the network.